tests/github-glue.test.ts

```diff
@@ -10,26 +10,26 @@ branch to be deleted on github and the local repo.  The test will
 attempt cleanup of a previously failed test.
 
 Setup:
-gitgitgadget.githubtest.githubuser must be configured for this test to
+gitgitgadget.githubTest.gitHubUser must be configured for this test to
 identify your GitHub login.
 The value must also be configured to identify a test repo to use as
-gitgitgadget.<login>.githubrepo.
+gitgitgadget.<login>.gitHubRepo.
 
 Additionally, a GitHub personal access token must be set for the
 login to succeed.  This can be restricted to the test repo.
 
 For example, these configuration settings are needed (where
-`octokity` is your GitHub login):
-git config --add gitgitgadget.githubtest.githubuser octokity
-git config --add gitgitgadget.octokity.githubrepo ggg-test
-git config --add gitgitgadget.octokity.githubtoken feedbeef...
+`octo-kitty` is your GitHub login):
+git config --add gitgitgadget.gitHubTest.gitHubUser octo-kitty
+git config --add gitgitgadget.octo-kitty.gitHubRepo ggg-test
+git config --add gitgitgadget.octo-kitty.gitHubToken feedbeef...
 
 The test repo must exist in github and locally.  It is expected to be
 located at the same directory level as this project (ie ../).
 */
 
 class GitHubProxy extends GitHubGlue {
-    public octo: Octokit | any;
+    public octo: Octokit;
     public constructor(workDir?: string, repo = "git") {
```

Comment:
You aren't getting name reuse warnings?  Will double check where they are coming from.

Reply:
No, I don't...

Reply:
Sorry, must have been a tslint warning.  Sigh.